src/api/event-info/index.ts

```diff
@@ -7,8 +7,7 @@ export interface EventInfo {
   // district "display_name" from TBA
   fullDistrict?: string
   locationName: string
-  lat?: number
```

Comment:
You can keep these on here, right? Cuz they still exist on the response, right?

Reply:
I suppose, I can't really think of anything they're used for though, and once this is fully deployed we could remove them from the BE - I left them in the BE response so we could have a smooth transition. Do you want to keep them around for something in the future?

Reply:
It is being used for sorting events by distance when 2 events are at the same time, if they both have locations.

Reply:
alright, adding back now. How did it pass the CI linter/typecheck?

Reply:
I had the same question so I dug into it. It passes types because the event comparer essentially requires "2 objects that have start/end dates and optionally lat/lon" (because many offseasons don't have locations). An `EventInfo`, without `lat/lon`, satisfies that.

It's a bummer but I guess it is what is expected from the type system. It _works_ without `lat`/`lon` so the types don't jump out to tell you that it's not going to work.